instances/technique/beta-galactosidaseStaining.jsonld

```diff
@@ -0,0 +1,15 @@
+
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/beta-galactosidaseStaining",
```

Comment:
change to: 
https://openminds.ebrains.eu/instances/technique/betaGalactosidaseStaining
(without the "-")